cmd/recv.go

```diff
@@ -26,7 +25,6 @@ func recvCommand() *cobra.Command {
 	}
 
 	cmd.Flags().BoolVarP(&verify, "verify", "v", false, "display verification string (and wait for approval)")
-	cmd.Flags().BoolVar(&hideProgressBar, "hide-progress", false, "suppress progress-bar display")
```

Comment:
@vu3rdd can you elaborate on what the issue was here?

Reply:
That makes sense, thanks. I think in the next iteration, we can use build tags to ensure we can get a wasm-compatible build without breaking existing functionality.